pkgs/development/tools/rust/cargo-rdme/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, rustPlatform, fetchCrate, stdenv, darwin }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-rdme";
+  version = "1.4.2";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-ZveL/6iWxnEz13iHdTjDA4JT29CbvWjrIvblI65XuMM=";
+  };
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.Security
```

Comment:
```suggestion
    Security
```

Reply:
I’m not sure what’s the ratoinale for this change though, could you explain it a bit?